static/webpack/webpack.prod.js

```diff
@@ -29,6 +31,14 @@ module.exports = () => {
           }
         }
       ]
-    }
+    },
+    optimization: {
+      minimizer: [
+        new ESBuildMinifyPlugin({
+          target: 'es5',
+          css: true
```

Comment:
does using this on top of cssnano shrink the css bundle even more? If not maybe we can get away with just using one of these?

Reply:
good point! I will try this with and without cssnano

Reply:
In my testing the esbuild css minimizer is 8% faster than CSS nano, and the resulting pages are 0.5% larger. Based on that, I think we can remove cssnano